quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusValidator.java

```diff
@@ -79,7 +79,7 @@ private void validateProperty(Property property) {
 			if (metadata == null) {
 				// Validate Unknown property
 				validateUnknownProperty(propertyName, property);
-			} else {
+			} else if (property.getValue() != null) {

```

Comment:
please remove this if condition and do the test inside validatePropertyValue which must manage validation type (as your PR) and required validation